docs/OpenTypeFeatureFileSpecification.md

```diff
@@ -3442,6 +3451,161 @@ the `vrt2` feature. The default vertical advance for these glyphs is the
+in the `name` table.
+
+```fea
+ElidedFallbackNameID <name ID>;
+```
+
+#### Design axes
+There can one or more design axes specified in the `STAT` table.
+
+```fea
+DesignAxis <axisTag> <axisOrdering num> {
+    name <name spec>;+
+};+
+```
+
+`axisTag` is a four-letter tag. `axisOrdering` is a signed number.
+
+The syntax for individual name string entries is the same as above.
+
+#### Axis values
+There can one or more axis values specified in the `STAT` table.
+
+```fea
+AxisValue {
+    flag <FLAG>+ ;
+    name <name string>;+
+    location ...;+
+};+
+```
+
+`flag` is optional, and defaults to 0 when omitted. Possible flags are
+`ElidableAxisValueName` and `OlderSiblingFontAttribute`, and more than one flag
+can be specified, e.g:
+
+
+```fea
+flag ElidableAxisValueName OlderSiblingFontAttribute;
+```
+
+There are several formats for `location` statement:
+
+##### location format A
+```fea
+location <axisTag> <value>;+
+```
+
+`axisTag` is a 4-letter tag and must correspond to one of the defined design
+axes in the table. `value` is signed number and can be specified using decimal,
+hex and octal formats as well (see §[9.e](#9.e)).
+
+There can be more than one `location` statement when this format is used (in
+this case the axis value will be `STAT` format 4 `AxisValue`, and format 1
+otherwise).
+
+##### location format B
+```fea
+location <axisTag> <nominalValue> <rangeMinValue> - <rangeMaxValue>;
+```
+
+Format for `axisTag` and the other values as above, and there must be space
+before the `-` to distinguish it from possible minus sign for
+`<rangeMaxValue>`.
+
+There can be only one `location` statement when this format is used (the axis
+value will be `STAT` format 2 `AxisValue`).
+
+##### location format C
+```fea
+location <axisTag> <value> <linkedValue>;
+```
+
+Format for `axisTag` and the other values as above.
+
+There can be only one `location` statement when this format is used (the axis
+value will be `STAT` format 3 `AxisValue`).
+
+#### Example
+```fea
+table STAT {
+   ElidedFallbackName { name "Regular"; };
+
+   DesignAxis wght 0 { name "Weight"; };
+   DesignAxis ital 1 { name "Italic"; };
+
+   # These examples are for illustrative purposes only;
+   # they won’t all be in a single STAT table:
+
+   # format 1
+   AxisValue {
+      location wght 400;
+      name "Regular";
+      name 3 1 0x411 "\5B9A\671F\7684";
+      flag ElidableAxisValueName;
+   };
+
+   # format 2
+   AxisValue {
+      location wght 400 300- 500;
+      name "Regular";
+      flag ElidableAxisValueName;
+   };
+
+   # format 3
+   AxisValue {
+      location wght 400 700;
+      name "Regular";
+      flag ElidableAxisValueName;
+   };
+
+   # format 3
+   AxisValue {
+      location ital 0 1;
+      name "Regular";
+      flag ElidableAxisValueName;
+   };
+
+   # format 4
+   AxisValue {
+      location wght 500;
+      location ital 1;
+      name "Mediumitalic";
```

Comment:
Totally doesn't matter at all, but it bothers me that the **i** is lowercase. Could it be `MediumItalic`?